client/src/components/RunCard/RunCard.js

```diff
@@ -76,24 +76,32 @@ const MapContainer = styled.div`
 const RunCard = ({
```

Comment:
It's very similar to `BigRunCard`. Maybe you can reduce it to a single component or make use of [compositions](https://reactjs.org/docs/composition-vs-inheritance.html).

Reply:
Made it to a part of my refactoring issue